WordPress/src/main/res/values/strings.xml

```diff
@@ -1733,7 +1733,7 @@
     <string name="me_disconnect_from_wordpress_com">Log out of WordPress.com</string>
 
     <!--TabBar Accessibility Labels-->
-    <string name="tabbar_accessibility_label_my_site">My Site. View your site and manage it, including stats.</string>
+    <string name="tabbar_accessibility_label_my_site">My Site. View your site andR.id.row_logout manage it, including stats.</string>
```

Comment:
This seems odd?

Reply:
Umm I'm not sure what made this change, I'll refactor